railties/lib/rails/generators/rails/sessions/templates/controllers/sessions_controller.rb

```diff
@@ -1,5 +1,5 @@
 class SessionsController < ApplicationController
-  allow_unauthenticated_access
+  allow_unauthenticated_access(only: %i[new create])
```

Comment:
👍

It is a style preference but to me this looks cleaner:
`allow_unauthenticated_access only: %i[ new create ]`

Reply:
Agree. Updated. Now it will be consistent with [this](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb.tt#L3)